lib/MusicBrainz/Server/Controller/Tag.pm

```diff
@@ -21,6 +21,16 @@ sub _load
     return $c->model('Tag')->get_by_name($name);
 }
 
+sub no_tag_provided : Path('/tag')
+{
+    my ($self, $c, $name) = @_;
+
+    # If no tag is passed, redirect to the tag cloud
+    $c->response->redirect(
+        $c->uri_for_action('/tag/cloud'));
+    $c->detach;
```

Comment:
You should be able to just do `$c->detach('/tag/cloud');` here and remove the above line.

Reply:
Never mind, it's definitely more useful if it performs a redirect.